recipes/tensorflow-lite/all/conandata.yml

```diff
@@ -1,13 +1,26 @@
 sources:
+  "2.14.0":
+    url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.14.0.tar.gz"
+    sha256: "ce357fd0728f0d1b0831d1653f475591662ec5bca736a94ff789e6b1944df19f"
   "2.12.0":
     url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.10.0.tar.gz"
     sha256: "b5a1bb04c84b6fe1538377e5a1f649bb5d5f0b2e3625a3c526ff3a8af88633e8"
   "2.10.0":
     url: "https://github.com/tensorflow/tensorflow/archive/refs/tags/v2.10.0.tar.gz"
     sha256: "b5a1bb04c84b6fe1538377e5a1f649bb5d5f0b2e3625a3c526ff3a8af88633e8"
 patches:
+  "2.14.0":
+    - patch_file: "patches/disable_fetch_content.patch"
+      patch_description: "Fail if the CMake build script tries to fetch external dependencies"
+      patch_type: "conan"
```

Comment:
This patch can probably be replaced with simply setting [`FETCHCONTENT_FULLY_DISCONNECTED`](https://cmake.org/cmake/help/latest/module/FetchContent.html#variable:FETCHCONTENT_FULLY_DISCONNECTED).